src/server/components/TidligereHistorikk.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import {
+  EPeriodetype,
+  ITidligereVedtaksperioder,
+  periodetypeTilTekst,
+} from '../../typer/dokumentApi';
+import { formaterIsoDato, formatterBooleanEllerUkjent } from '../utils/util';
+
+export const TidligereHistorikk: React.FC<{
+  tidligereVedtaksperioder: ITidligereVedtaksperioder | undefined;
+}> = ({ tidligereVedtaksperioder }) => {
+  const periodeHistorikkOvergangsstønad =
+    tidligereVedtaksperioder?.sak.periodeHistorikkOvergangsstønad;
+
+  const TidligereHistorikkOvergangsstønadTabell: React.FC = () => {
+    if (!periodeHistorikkOvergangsstønad || periodeHistorikkOvergangsstønad?.length < 1)
+      return <></>;
+
+    return (
+      <table>
+        <tr>
+          <th>Periode</th>
+          <th>Periodetype</th>
+          <th>Måneder med utbet.</th>
+          <th>Måneder uten utbet.</th>
+        </tr>
+        {periodeHistorikkOvergangsstønad?.map(periode => {
+          return (
+            <tr>
+              <td>
+                {formaterIsoDato(periode.fom)} - {formaterIsoDato(periode.tom)}
+              </td>
+              <td>{periodetypeTilTekst[periode.vedtaksperiodeType] || ''}</td>
+              <td>{periode.antallMåneder}</td>
+              <td>
+                {periode.antallMånederUtenBeløp >= 1 &&
+                periode.vedtaksperiodeType !== EPeriodetype.SANKSJON
+                  ? periode.antallMånederUtenBeløp
+                  : '-'}
+              </td>
+            </tr>
+          );
+        })}
+      </table>
+    );
+  };
```

Comment:
Vi har ikke definert egne React-komponenter på dette viset ellers i kodebasen. Så her vil jeg anbefale at du flytter `TidligereHistorikkOvergangsstønadTabell` nederst i filen under `TidligereHistorikk`-komponenten. Dette er på grunn av to ting:

1. Da blir koden likere sånn vi ellers gjør det i kodebasen
2. Jeg synes det blir enklere å lese koden

Reply:
Enig at det blir mer oversiktlig, har gjort det nå. Årsaken til at jeg har tatt det ut i en komponent er at det på sikt kan blir flere tabeller. Tenkte derfor at det ville være en god ting å bryte opp koden.